### ISSUE TITLE
Windows only: Atom command-line throws `find: `ParentProcessId': No such file or directory` error.

### ISSUE THREAD

**Issue by dougireton** [atom/atom#11270]
### Description

Windows only: Atom command-line throws `find: `ParentProcessId': No such file or directory` error.

### Steps to Reproduce
This bug only shows up on Windows. It works as expected on Mac and Linux.
We reproduced this on a Windows 8.1 box.

1. From the command line, run atom with any switches, e.g. `atom -v`.

**Expected behavior:**
Version string, e.g. `1.6.0`

**Actual behavior:**

```
atom -v
```

```
find: `ParentProcessId': No such file or directory
'ELSE' is not recognized as an internal or external command,
operable program or batch file.

1.6.0
```

### Versions
Atom 1.6.0.
This doesn't happen in Atom 1.5.4.

**Comment by tzhbami7** [comment]
:+1: having the same issue on Windows 10

**Comment by damieng** [comment]
This has been fixed for the next release - 1.7.0 - alternatively if you want it sooner copy the atom.cmd file from here and replace the one in your %appdata%\atom\app-1.6.0\resources\cli folder.

https://github.com/atom/atom/blob/master/resources/win/atom.cmd
